agent/engine/engine_windows_integ_test.go

```diff
@@ -55,7 +50,11 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/taskresource"
 	taskresourcevolume "github.com/aws/amazon-ecs-agent/agent/taskresource/volume"
 	"github.com/aws/amazon-ecs-agent/agent/utils"
+
 	"github.com/aws/aws-sdk-go/aws"
+
```

Comment:
"github.com/aws/aws-sdk-go/aws" and the next block are all third party imports right? so I think L55 extra line is not needed.